scripts/generate_ssl_cert

```diff
@@ -52,7 +52,7 @@ do
 	echo "DNS.${i}=${x}" >> config
 done
 
-openssl genrsa 1024 > privkey.rsa
+openssl genrsa 2048 > privkey.rsa
```

Comment:
2048 is good, but the requirements mention 3072. Which one is correct?

Reply:
This is why I mentioned the comments - CA-296204 details why 3072 failed - it involves a static length buffer.

Reply:
Yes. We are now confirmed to use 2048. I will change the requirement.